doc/source/changes/version_0_29.rst.inc

```diff
@@ -175,73 +110,155 @@ New features
+    a  a0  a1  a2
+        3   6   2
+
+  Generate values following a uniform distribution
+
+    >>> uniform(axes='a=a0..a2')
+    a                   a0                  a1                  a2
+       0.33293756929238394  0.5331412592583252  0.6748786766763107
+
+  Generate values following a normal distribution (:math:`\mu` = 1 and :math:`\sigma` = 2 in this example)
+
+    >>> normal(1, scale=2, axes='a=a0..a2')
+    a                   a0                 a1                  a2
+       -0.9216651561025018  5.119734598931103  4.4467876992838935
+
+  Randomly shuffle an existing array along one axis
+
+    >>> arr = ndtest((3, 3))
+    >>> arr
+    a\b  b0  b1  b2
+     a0   0   1   2
+     a1   3   4   5
+     a2   6   7   8
+    >>> permutation(arr, axis='b')
+    a\b  b1  b2  b0
+     a0   1   2   0
+     a1   4   5   3
+     a2   7   8   6
+
+  Generate values by randomly choosing between specified values (5, 10 and 15 in this example), potentially with a
+  specified probability for each value (respectively a 30%, 50%, 20% probability of occurring in this example).
+
+    >>> choice([5, 10, 15], p=[0.3, 0.5, 0.2], axes='a=a0,a1;b=b0..b2')
+    a\b  b0  b1  b2
+     a0  15  10  10
+     a1  10   5  10
+
+  Same as above with labels and probabilities given as a one dimensional LArray
+
+    >>> proba = LArray([0.3, 0.5, 0.2], Axis([5, 10, 15], 'outcome'))
+    >>> proba
+    outcome    5   10   15
+             0.3  0.5  0.2
+    >>> choice(p=proba, axes='a=a0,a1;b=b0..b2')
+    a\b  b0  b1  b2
+     a0  10  15   5
+     a1  10   5  10
+
+* made a few useful constants accessible directly from the larray module: ``nan``, ``inf``, ``pi``, ``e`` and
+  ``euler_gamma``. Like for any Python functionality, you can choose how to import and use them. For example, for
+  ``pi``:
+
+    >>> from larray import *
+    >>> pi
+    3.141592653589793
+    OR
+    >>> from larray import pi
+    >>> pi
+    3.141592653589793
+    OR
+    >>> import larray as la
+    >>> la.pi
+    3.141592653589793
+
+* added :py:obj:`Group.equals()` method which compares group names, associated axis names and labels between two groups:
+
+    >>> a = Axis('a=a0..a3')
+    >>> a02 = a['a0:a2'] >> 'group_a'
+    >>> # different group name
+    >>> a02.equals(a['a0:a2'])
+    False
+    >>> # different axis name
+    >>> other_axis = a.rename('other_name')
+    >>> a02.equals(other_axis['a0:a2'] >> 'group_a')
+    False
+    >>> # different labels
+    >>> a02.equals(a['a1:a3'] >> 'group_a')
+    False
 
-.. _misc:
 
 Miscellaneous improvements
 --------------------------
 
-* saving or loading a session from a file now includes `Axis` and `Group` objects in addition to arrays
-  (closes :issue:`578`):
+* completely rewritten the 'Load And Dump Arrays, Sessions, Axes And Groups' section of the tutorial
+  (closes :issue:`645`)
+
+* saving or loading a session from a file now includes :ref:`api-axis` and :ref:`api-group` objects in addition to
+  arrays (closes :issue:`578`).
```

Comment:
s/)./):/

Reply:
I prefer not. I don't like a : followed by another title.